Define and activate AWS 11.3.2 release (#304)
giantswarm/releases: release-notes/aws/v11.3.2.md

## :zap: Giant Swarm Release v11.3.2 for AWS :zap:

**If you are upgrading from 11.3.1, upgrading to this release will not roll your nodes. It will only update the apps.**

This release fixes an external-dns issue affecting clusters in AWS China. We added an option for the app to use CNAME records instead of ALIAS, in order for DNS resolution to work in China.

> Note though that the fix works for new cluster creation only. For cluster upgrades, there is a known upstream external-dns issue: Existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred.

Reviewer: ```suggestion
Note that the fix works for new cluster creation only. Upgrades require a manual workaround. We're working with the upstream external-dns community to fix it. In the meantime, we recommend delegating cluster upgrades to your SE.```

> Note though that the fix works for new cluster creation only. For existing cluster upgrades there's known upstream issue in external-dns - existing ingress A+TXT record sets are not being replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred.

Reviewer: ```suggestion
Note though that the fix works for new cluster creation only. For cluster upgrades, there is a known upstream external-dns issue: Existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred.
```

> We're working with external-dns community to fix the issue for cluster upgrades. Until then, if you're upgrading a cluster in the AWS 11.0.1 to 11.3.1 range, a manual workaround has to be applied: Delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets. **It is recommended to delegate this to your SE.**

Reviewer: ```suggestion
**Note for SEs:** When upgrading a cluster, existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets.```

> We're working with external-dns community to provide the automatic working solution for cluster upgrades too. Until then, if you're upgrading cluster from aws platform release in [11.0.1 - 11.3.1] range, a manual workaround has to be applied - make sure to delete ingress A+TXT record sets, external-dns will then automatically create CNAME+TXT record sets. It is recommended to delegate applying this workaround to your SEs.

Reviewer: ```suggestion
We're working with external-dns community to fix the issue for cluster upgrades. Until then, if you're upgrading a cluster in the AWS 11.0.1 to 11.3.1 range, a manual workaround has to be applied: Delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets. **It is recommended to delegate this to your SE.**
```
